.circleci/config.yml

```diff
@@ -134,7 +134,7 @@ commands:
     steps:
       - restore_cache:
           keys:
-            - v3-gem-cache-{{ checksum "Gemfile.lock" }}-{{ arch }}
+            - v4-gem-cache-{{ checksum "Gemfile.lock" }}-{{ arch }}
```

Comment:
I changed the "version" of the cache to prevent using the previous v3 one which didn't contain the Ruby 3.2.0 installation